onnxruntime/core/graph/contrib_ops/bert_defs.cc

```diff
@@ -1059,7 +1058,7 @@ ONNX_MS_OPERATOR_SET_SCHEMA(
                 "kv_sequence_length.",
                 "T")
         .TypeConstraint("T", {"tensor(float16)"}, "Constrain input and output to float tensors.")
-        .TypeConstraint("M", {"tensor(int32)", "tensor(int64)"}, "Constrain past sequence length to int tensor.")
+        .TypeConstraint("M", {"tensor(int64)"}, "Constrain past sequence length to int tensor.")
```

Comment:
update the description since there is no past sequence length in input now. Check also the doc for operator.